src/pages/SearchPage/index.jsx

```diff
@@ -0,0 +1,179 @@
+   * @type {propTypes}
+   *
+   * @param {Object} props - React PropTypes
+   *
+   * @property {string} term - search query
+   * @property {string} query - user input
+   * @property {boolean} searched - to if the user is searched
+   * @property {Function} formSubmit - handle term
+   * @property {Function} handleInputChange - handle input change
+   * @property {Function} handleTermChange - handle term
+   * @property {Function} handleSearch - process the search
+   * @property {Function} handleIncomingSearch - handle the search on the global header
+   * @property {Function} renderList - render searched item
+   *
+   */
+  static propTypes = {
+    search: PropTypes.instanceOf(Object).isRequired,
+    searchArticlesRequest: PropTypes.func.isRequired,
+    history: PropTypes.instanceOf(Object).isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    const {
+      history: {
+        location: { search },
+      },
+    } = this.props;
+    const query = search.split('=')[1];
+    this.state = {
+      term: 'tag',
+      query,
+      searched: false,
+    };
+  }
+
+  componentDidMount() {
+    const { query } = this.state;
+    if (query) {
+      this.handleIncomingSearch();
+    }
+  }
+
+  submitForm = (term) => {
+    const { query } = this.state;
+    const { searchArticlesRequest: searchArticles } = this.props;
+    searchArticles({ term, query });
+  };
+
+  handleInputChange = ({ target: { value } }) => {
+    this.setState({ query: value });
+  };
+
+  handleTermChange = ({ target: { value } }) => {
+    this.setState({ term: value }, this.submitForm(value));
+  };
+
+  handleSearch = ({ keyCode }) => {
+    const { term } = this.state;
+    if (keyCode === 13) {
+      this.setState({ searched: true }, this.submitForm(term));
+    }
+  };
+
+  handleIncomingSearch = () => {
+    const { term } = this.state;
+    this.setState({ searched: true }, this.submitForm(term));
+  };
+
+  renderList = () => {
+    const {
+      search: { data }
+    } = this.props;
+
+    return (data)
+      && data.map((article) => (
+        <Link
+          to={`/articles/${article.slug}`}
+          key={article.ArticleId}
+          className="card-display"
+        >
+          <Card
+            img={article.coverImageUrl}
+            tag={article.category}
+            title={article.title}
+            summary={article.summary}
+            author={article.author.name}
+          />
+        </Link>
+      ));
+  };
+
+  render() {
+    const {
+      search: { loading },
+    } = this.props;
+    const { query, searched } = this.state;
+
+    return (
+      <div className="search-container">
+        <div>
+          <Search
+            searchkeyUp={this.handleSearch}
+            changeValue={this.handleInputChange}
+            searchValue={query}
+            required
+          />
+        </div>
+        {loading && <Spinner caption="Searching" />}
+        {searched && (
+          <div>
+            <button
```

Comment:
Create a reuseable component called `LinkButton` which can be used to render items that look like links but do not have `href`s. And then used it for each query item. I gave this feedback during our live review, I wonder why it wasn't implemented.